webapp/tests/karma/ts/modules/contacts/contacts.component.spec.ts

```diff
@@ -613,14 +627,11 @@ describe('Contacts component', () => {
 
   describe('last visited date', () => {
     it('does not enable LastVisitedDate features not allowed', () => {
-      expect(authService.has.callCount).equal(2);
-      expect(authService.has.args[0]).to.deep.equal(['can_view_last_visited_date']);
-      expect(authService.has.args[1]).to.deep.equal(['can_view_old_filter_and_search']);
```

Comment:
Because the other components are now being initialised there are more like 6 auth service has requests. I'm not sure if there's much value in checking the call count any more. What do you think?